v2/samples/network/v1api20220701/v1api20220701_applicationgateway.yaml

```diff
@@ -0,0 +1,148 @@
+      DgQIdj1kY7PLoDkCAggABIIEyG0HEW9aIMrFT4F2MuBqY6Nu981+dRuLUh6wjWbh
+      Vkug5y3e+bsH7/kl9aWQjFzwMmlverxhVISu/MJ4wiACZXuGWvGk+jDRUBrkSr7v
+      2ZbWMMaShELSTpbvXO6BDSWA6yInT4BeZYrFKOAysdtXHhPOs/nBF1FOagZnXW+Y
+      SxSc/oCQgiflEotmo6LmoTxE3Ta0fTZTXlb5xmnbBEt+30S6sS+lU1nllEgvpNgG
+      6XWxugGQsKAyYagxqQzEYQJ+tytMc1pntURsVxsl41xbceN1hDxfrPnLfZ6XUKsq
+      rKUpZdxN0WVHErdL58JJzuoIswtnsB4+5hMFWgY4J7hJBE+r4/oQ7S6w8gHqYu+p
+      aBkdb8EqdCh1X6sXCzUBOpDAAqzkPiAhIzF27zUZ+1F1mWlcv8/CRTh6TdQJTwaH
+      tRPptTR/rLEmN6nBVhNw/tYVxY/Y93vuO79HdFtFl5UJqdYRjOmf6QTJU9thO7wy
+      Qj7ZEoEMYsg17qofz2DEsxdB7UahOJQ3P7pD3nAL8QT3bwNeF3f1hOnJF736Ea6q
+      gxvmXd653T87QRisjYrbssVQ0tDj0l44lDyllFSV9sbopGiuuqGctCCUUpKGZeTV
+      gUTZs8lOGe/ACspOD/tsQnFkOa/FY2AxqSzk4t7p62nPeabu+ZOzQiYmB+nxrju/
+      sAjpdQqF866OEYUxisobL8S0RUXRYZtevMKCS09PHJ8Gv5bYHR8xdsGswBuo2Cxh
+      OWoW22Y7zkdn4YwVci3tqbZ+c3SeD1Aw0nVz8uvxljHSphed2aNDDv+WlVTQv2BF
+      lH6ONKp6hOIVWDLEVzbN3rHbMYFaz2o+TZZyzSGXK5THjAxlM46w9OEYqmQdULIt
+      nZSB6qF31buMPjvRzt4NHlDp/cQD/Onc831eJYA4GP3vAfxZC2PYddS1kK+42oZc
+      1+IWQyI66xitQN83eJ4zYAgkhMbCGM2bSkM87WOoHh9GZb/qo8urx32llGFRALLL
+      98aIadMJZifCdBc0g5FF7TPwzhq90tIvfA3F9Ha9wSCmyHFm2EI+jLxsjD+zIkKQ
+      2tkXeICKZeVofVNhe/s2PKqMHj5dbdaPkOLGTZFfiCFx6JYVLFaGePWixICgWbFc
+      djpzGLXndeNnMJoHhhW/hRQVzktVjOcwqbyL223+Tg07VTZR13xh//p54c9J+/ic
+      42OyPNtf3lVV/3lLE65drgKViwgRh8soifl2ycYakwr1n53wntNjfzohXGkoIFzX
+      llt+OgUnl6X9QWK0/eq8i1pqKcy2nJkiGrIjlCgR4ubPJBUMbt3VZXjK9NJX5Oe3
+      eSIJu3aU7oIci+f8zXfSiivgKHBkyk7Z3AOq1RPBwPlqQlqneJD3HP7JhiRQMB/a
+      NoKTGbz3Py6GfdaSIxkNkutT63KQe0Auj9/0oMFrtqJaTT+Zu88o2A1aRnADvEwS
+      uob+N/7mQsQ/hUVWQN7OdxGU+JyuEDM8KWzpFlocDBuaeN654F4YIKdfxvjB4ITP
+      LRs5Xo3N2OBFIlLkOtjlPTzZabAqUynJ4LPfJCbH078SfY4YACGcnUcMHZWC5ueI
+      v7ZY4ZA8xxg4ajdspj2n8HuvugU4k2GExXSxpDaJctaA/XnBMnWrUFo0CTElMCMG
+      CSqGSIb3DQEJFTEWBBQ1FOpSKUDiMNgnn3GIeT1TJ7imnzAxMCEwCQYFKw4DAhoF
+      AAQUZcDhMEBZrBZVLOTMV1dEBK7cxTcECIAwrio+tqKLAgIIAA==
+  sslProfiles:
+  - name: app-gw-ssl-profile-1
+    sslPolicy:
+      policyName: AppGwSslPolicy20220101S
+      policyType: Predefined
+  sslPolicy:
+      policyName: AppGwSslPolicy20220101S
+      policyType: Predefined
+  webApplicationFirewallConfiguration:
+    enabled: true
+    firewallMode: Detection
+    ruleSetType: OWASP
+    ruleSetVersion: 3.2
+  backendAddressPools:
+  - name: app-gw-be-pool-1
+    backendAddresses:
+    - IpAddress: 10.0.1.1
+  backendHttpSettingsCollection:
+  - name: app-gw-be-pool-1
+    port: 8443
+    protocol: Https
+    cookieBasedAffinity: Disabled
+    pickHostNameFromBackendAddress: false
+  httpListeners:
+  - name: app-gw-http-listner-1
+    frontendIPConfiguration:
+      reference:  
+        armId: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/<rgName>/providers/Microsoft.Network/applicationGateways/<appgatewayname>/frontendIPConfigurations/<feIpConfigName>
```

Comment:
You can replace `<rgname>` with `aso-sample-rg` sincethat's the sample rg which the sample is written in the context of.
Same with `appgatewayname`, it can be `aso-sample-application-gateway`, the name of this application gateway. Use the actual `feIpCOnfigName` too.

Same for all the refs below as well.

Reply:
fixed